### DISCUSSION TITLE
Issue with the latest firmware version

### DISCUSSION THREAD

**Discussion by gd35** [tbnobody/OpenDTU#386] [General]
I just upgraded the firmware with the latest version. I did not do it for a while. However, since the upgrade, the reporting of data is not done on a regular basis (with OpenHAB):

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/6339942/204085419-e2e8a63f-f217-42a5-ae60-ff19f4c204a5.png">

Left side: with the old version of the firmware
Right side: with the new one

The live data show this:

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/6339942/204085458-84b9182d-ab14-4ee1-aeb1-c6f128ded7b8.png">

It seems that the Data Age is quite high. The hardware configuration is the same. I did not change anything except the firmware upgrade

Do you have any idea about the issue ?

**Comment by tbnobody** [comment]
Do you know from which version you where coming from? without  this its quite hard to determine the changes. But the most changes in the last days/weeks where interface related. They had nothing to do with the fetching of the inverter itself.

**Reply by gd35** [reply]
I did not remember exactly... it was a version published in September when  I started to build the device. But I did a little progress on that issue since my first post. I have had to do the configuration again after the upgrade. I just changed the DTU setting (Maximum for the PA Level, it was High before). Then, the data age is decreasing. So I suspect it is a reception problem. It is time to buy a new nrf24L01+ module with an antenna ! I like very much the new firmware version. Thanks for your help

**Comment by tbnobody** [comment]
The config re-do issue has to do with this: https://github.com/tbnobody/OpenDTU/discussions/285

Regarding another RF module. It seems that modules from ebyte are much more reliable
e.g. something like this: https://www.amazon.de/Ebyte-E01-ML01DP5-Transmitter-Empf%C3%A4nger-nRF24L01P/dp/B07P8TTHZH/

They are also pin compatible (you maybe need a 2,4ghz antenna)

**Reply by gd35** [reply]
Amazing, this module was already in my amazon basket shop before replying to your message. I recently bought an ebyte CC1101 module to receive data from my water meter for the same reason. And yes, they ebyte modules are much much better!. Thanks.

**Reply by gd35** [reply]
Just received the new module with its antenna. It works like a charm.
Thanks.
